torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -142,69 +142,35 @@ def write_header(self):
                 os.path.join(os.path.dirname(__file__), "aoti_runtime", "interface.cpp")
             ) as f:
                 self.header.splice(f.read())
-        else:
             self.header.splice(
+                f"""
+                #include <torch/csrc/inductor/aoti_runtime/arrayref_tensor.h>
```

Comment:
FYI, `arrayref_tensor.h` and `thread_local.h` should also conditional depends on whether we turn on the array-ref mode. Refactor them out should also help to cut the compilation time.